tests/ResarchStation/Domain/SpaceResearchStation/SpaceResearchStationTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\ResarchStation\Domain\SpaceResearchStation;
+
+use App\DomainModel\EarthScientistDomain;
+use App\DomainModel\SpaceResearchStation;
+use App\DomainModel\SpaceScientist;
+use App\Exception\WrongScientistTypeException;
+use PHPUnit\Framework\TestCase;
+use function count;
+
+class SpaceResearchStationTest extends TestCase
+{
+    private SpaceResearchStation $researchStation;
+
+    /** @test */
+    public function try_to_add_wrong_scientist_type_should_throw_error(): void
+    {
+        $this->expectException(WrongScientistTypeException::class);
+
+        $this->researchStation = new SpaceResearchStation(
```

Comment:
Jeżeli dwa razy twój SpaceResearchStation wyglada tak samo to mozesz go utworzyc w setUp.